dd-java-agent/instrumentation/jedis-1.4/src/test/groovy/JedisClientTest.groovy

```diff
@@ -84,10 +86,10 @@ class JedisClientTest extends AgentTestRunner {
       trace(1, 1) {
         span(0) {
           serviceName "unnamed-java-app"
-          operationName "redis.query"
-          resourceName "GET"
+          operationName "redis.query" 
```

Comment:
is the resource name no longer `GET`?

Reply:
It is. I initially took out all `resourceName`s, since I wasn't sure we should be testing spans for them once they are filtered out. I omitted putting  this back.